### ISSUE TITLE
`zfs-utils` CI Builds are Failing

### ISSUE THREAD

**Issue by prikhi** [archzfs/archzfs#521]
Started about 5 days ago: https://ci.archzfs.com/#/builders/13/builds/2455
Not sure what's wrong, maybe related to this line?

> Failed to open system bus: No such file or directory

**Comment by Ivan-Johnson** [comment]
@minextu, this seems to be a recurring issue (2x in #498, 3x in #445, 1x #422, and probably more that didn't turn up in my quick search). [Back in August](https://github.com/archzfs/archzfs/issues/498#issuecomment-1667047903) you mentioned that you planned on migrating to a new server to help with this issue. Has any progress been made on that front? Am I correct in assuming that that would only reduce the frequency with which manual interventions are necessary, and not eliminate manual interventions entirely?

I'm willing to donate ~20 hours of my time if you think it would help eliminate manual interventions entirely. In my day job I almost exclusively code in C and I have never even looked at the `archzfs` codebase before, but I like to think I know a thing or two about Bash after using it as my shell for the past ~10 years. Would you be interested in working together on this?

**Comment by lightdot** [comment]
@minextu, perhaps you'd rather do all the work yourself then delegate, that's fine and completely understandable. But if not, I'd also like to offer help.

I've been maintaining various packages for a long time now (going back to Red Hat, pre-Fedora days) and currently do so for a few in AUR. I generally know what I'm doing, if not, I tend to figure it out.

These days I'm tempted to update archzfs for myself, but that would feel like work wasted. I'd rather contribute publicly...

**Comment by minextu** [comment]
Sorry I've been a bit busy these days, so everytime something breaks it takes me a bit to find time to fix it. 

I'm always open for help, but currently the issue is not disk space, its a bit more complex: The build script in this repository is using clean-chroot-manager to build everything in a clean environment, it uses systemd-nspwan under the hood. Now the ci project (archzfs/archzfs-ci) is buildbot inside docker containers, which recently started having issues running systemd-nspawn inside docker. ```Failed to open system bus: No such file or directory```

Long story short, we either need to fix that systemd issue or migrate away from clean-chroot-manager or stop using docker / buildbot.

If should have some time after new years to look into it, or at least manually start a build process. Maybe someone has some time / experience with this to help?

**Comment by minextu** [comment]
I pushed a manual update, packages are up to date for now

**Comment by minextu** [comment]
CI Environment has been fixed. Thanks a lot to @UweSauter and @techmunk for investigating and fixing the issue!
